### PR TITLE
Roll dart, remove patched_sdk from flutter now that frontend handles patching

### PR DESCRIPTION
**Pull request by aam**

### CODE REVIEW

**Comment by aam** [comment]
cc @rmacnak-google

File: lib/snapshot/BUILD.gn [hunk 1]
```diff
@@ -414,10 +308,6 @@ action("compile_non_strong_platform") {
 
   inputs = []
```
Comment (rmacnak-google) [inline lib/snapshot/BUILD.gn]: Need libraries.json here as well.

File: lib/snapshot/BUILD.gn [hunk 2]
```diff
@@ -445,10 +335,6 @@ action("compile_platform") {
...
```
Comment (rmacnak-google) [inline lib/snapshot/BUILD.gn]: Need libraries.json here as well.

Comment (aam) [reply lib/snapshot/BUILD.gn]: Done, thanks!

File: lib/snapshot/BUILD.gn (hunk 1 above)
Comment (aam) [reply lib/snapshot/BUILD.gn]: Done, thanks!

File: lib/snapshot/BUILD.gn [hunk 3]
```diff
@@ -433,20 +325,18 @@ action("compile_platform") {
...
-  
+
   sources = [
-    "$root_out_dir/flutter_patched_sdk/lib/libraries.json",
+    "libraries.json",
   ]
 
   outputs = [
     "$root_out_dir/flutter_patched_sdk/platform_strong.dill",
     "$root_out_dir/flutter_patched_sdk/vm_outline_strong.dill",
```
Comment (a-siva) [inline lib/snapshot/BUILD.gn]: Do we still need these vm_outline dill files?

**Pull request merged**